corehq/apps/api/tests/lookup_table_resources.py

```diff
@@ -0,0 +1,262 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+import json
+
+from corehq.apps.api.tests.utils import APIResourceTest
+from corehq.apps.fixtures.models import (
+    FieldList,
+    FixtureDataItem,
+    FixtureDataType,
+    FixtureTypeField,
+)
+from corehq.apps.fixtures.resources.v0_1 import (
+    LookupTableItemResource,
+    LookupTableResource,
+)
+
+
+class TestLookupTableResource(APIResourceTest):
+    resource = LookupTableResource
+    api_name = 'v0.5'
+
+    def setUp(self):
+        super(TestLookupTableResource, self).setUp()
+        self.data_type = FixtureDataType(
+            domain=self.domain.name,
+            tag="lookup_table",
+            fields=[
+                FixtureTypeField(
+                    field_name="fixture_property",
+                    properties=["lang", "name"]
+                )
+            ],
+            item_attributes=[]
+        )
+        self.data_type.save()
+
+    def tearDown(self):
+        self.data_type.delete()
+        super(TestLookupTableResource, self).tearDown()
+
+    def _data_type_json(self):
+        return {
+            "fields": [
+                {
+                    "field_name": "fixture_property",
+                    "properties": ["lang", "name"],
+                },
+            ],
+            "item_attributes": [],
+            "id": self.data_type._id,
+            "is_global": False,
+            "resource_uri": "",
+            "tag": "lookup_table"
+        }
+
+    def test_get_list(self):
+        response = self._assert_auth_get_resource(self.list_endpoint)
+        self.assertEqual(response.status_code, 200)
+
+        fixture_data_types = json.loads(response.content)['objects']
+        self.assertEqual(len(fixture_data_types), 1)
+        self.assertEqual(fixture_data_types, [self._data_type_json()])
```

Comment:
Why doesn't this need to be `assertListEqual`? (Same for `TestLookupTableItemResource .test_get_list`)

Reply:
`assertEqual` gets mapped to the type appropriate equality comparison https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual (and you can add to that if you're subclassing `TestCase` / `SimpleTestCase`)

Reply:
did not know that (obviously), thanks